package.json

```diff
@@ -42,26 +42,24 @@
     "@babel/eslint-parser": "7.14.7",
     "@babel/preset-react": "^7.14.5",
     "@parcel/babel-preset-env": "^2.0.0-beta.2",
-    "@parcel/runtime-react-refresh": "^2.0.0-beta.2",
-    "@parcel/transformer-sass": "^2.0.0-beta.2",
+    "@parcel/runtime-react-refresh": "2.0.1",
+    "@parcel/transformer-sass": "2.0.1",
+    "@types/md5": "2.3.1",
     "eslint": "7.30.0",
-    "eslint-config-semistandard": "16.0.0",
-    "eslint-config-standard": "16.0.3",
-    "eslint-plugin-import": "2.23.4",
-    "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "5.1.0",
-    "eslint-plugin-react": "7.24.0",
-    "eslint-plugin-standard": "5.0.0",
-    "parcel": "^2.0.0-beta.2",
+    "eslint-plugin-import": "2.25.3",
+    "eslint-plugin-prettier": "4.0.0",
+    "eslint-plugin-promise": "5.1.1",
+    "eslint-plugin-react": "7.27.0",
+    "parcel": "2.0.1",
+    "prettier": "2.4.1",
     "standard-version": "9.3.0",
-    "stylelint": "13.13.1",
-    "stylelint-config-standard": "22.0.0"
+    "typescript": "4.4.4"
   },
   "scripts": {
     "build": "parcel build index.html",
     "release": "standard-version",
     "start": "parcel serve index.html",
-    "test": "eslint . && stylelint css",
```

Comment:
Eslint is not the same as prettier though. Prettier is just formatting

Reply:
Dooh!  Right you are.  Will fix.

Reply:
Holy fuck, trying to get vscode, eslint, prettier, typescript, react & JSX to all play nice ...
![image](https://media.giphy.com/media/OpesLQSjwdGj6/giphy.gif)

Reply:
Well, that wasn't any trouble at all 🙄